src/AbstractAlgebra.jl

```diff
@@ -203,6 +203,12 @@ import .PrettyPrinting: expressify
 #
 ###############################################################################
 
+function check_parent(a::T, b::T, throw::Bool = true) where T
```

Comment:
It seems requiring the same type is too strict in general; eg for matrix versus matrix view
```suggestion
function check_parent(a, b, throw::Bool = true)
```